app/src/sharedTest/java/org/oppia/android/app/options/ReadingTextSizeFragmentTest.kt

```diff
@@ -115,20 +119,37 @@ class ReadingTextSizeFragmentTest {
   @Test
   fun testTextSize_changeTextSizeToLarge_changeConfiguration_checkTextSizeLargeIsSelected() {
     launch<ReadingTextSizeActivity>(createReadingTextSizeActivityIntent("Small")).use {
-      checkTextSize(SMALL_TEXT_SIZE)
-      updateTextSize(LARGE_TEXT_SIZE)
+      verifyItemIsCheckedInTextSizeRecyclerView(SMALL_TEXT_SIZE_INDEX)
+      clickOnTextSizeRecyclerViewItem(LARGE_TEXT_SIZE_INDEX)
       rotateToLandscape()
-      checkTextSize(LARGE_TEXT_SIZE)
+      verifyItemIsCheckedInTextSizeRecyclerView(LARGE_TEXT_SIZE_INDEX)
+    }
+  }
+
+  @Test
+  fun testTextSize_checkTextSizeOfAllFourItems_textSizeMatchedCorrectly() {
```

Comment:
```
'TextView with text size: 50.4' doesn't match the selected view.
Expected: TextView with text size: 50.4
Got: "AppCompatTextView{id=2131296998, res-name=text_size_text_view, visibility=VISIBLE, width=125, height=68, has-focus=false, has-focusable=false, has-window-focus=true, is-clickable=false, is-enabled=true, is-focused=false, is-focusable=false, is-layout-requested=false, is-selected=false, layout-params=android.widget.LinearLayout$LayoutParams@af2015a, tag=null, root-is-layout-requested=false, has-input-connection=false, x=238.0, y=50.0, text=Small, input-type=0, ime-target=false, has-links=false}"
```

Failing for me on espresso.

Reply:
I have got following results with `Pixel 3 API 28` emulator
```
//      matchTextSizeOfTextSizeRecyclerViewItem(
//        // Expected = 39.600002
//        SMALL_TEXT_SIZE_INDEX, defaultTextSizeInFloat * SMALL_TEXT_SIZE_SCALE
//      )
//      matchTextSizeOfTextSizeRecyclerViewItem(
//        // Expected = 49.5
//        MEDIUM_TEXT_SIZE_INDEX, defaultTextSizeInFloat * MEDIUM_TEXT_SIZE_SCALE
//      )
//      matchTextSizeOfTextSizeRecyclerViewItem(
//        // Expected = 59.4
//        LARGE_TEXT_SIZE_INDEX, defaultTextSizeInFloat * LARGE_TEXT_SIZE_SCALE
//      )
//      matchTextSizeOfTextSizeRecyclerViewItem(
//        // Expected = 69.299995
//        EXTRA_LARGE_TEXT_SIZE_INDEX, defaultTextSizeInFloat * EXTRA_LARGE_TEXT_SIZE_SCALE
//      )
```

Reply:
put the solution below.